package.json

```diff
@@ -7,7 +7,7 @@
   "author": "Art.sy Inc <it@artsymail.com>",
   "license": "MIT",
   "engines": {
-    "node": "^10.13.0"
+    "node": "^12.14.0"
```

Comment:
What happens if this is bumped, with respect to Palette, Force, Volt (which I believe uses Reaction stuff), etc.?

Reply:
Actually, I'm going to remove this constraint so that the consuming apps can set the version.